src/institutions/Institution.jsx

```diff
@@ -71,11 +71,11 @@ const Institution = ({
   )
 }
 
-Institution.PropTypes = {
+Institution.propTypes = {
   institution: PropTypes.object,
   filing: PropTypes.object,
   submission: PropTypes.object,
-  submissions: PropTypes.object,
+  submissions: PropTypes.array,
```

Comment:
Not sure why this wasn't triggered before, but React started to warn that we were passing an array (which we are) instead of an object. Maybe just didn't notice it before.

Reply:
Wasn't warning before because of the incorrect capitalization ^^; Good catch